index.html

```diff
@@ -0,0 +1,172 @@
+          some of my work and experience! If you like what you see and have a
+          project you need coded, don’t hestiate to contact me.
+        </p>
+      </div>
+      <div class="icon-style">
+        <ul class="icons">
+          <li>
+            <a href="#"><img src="images/git-logo.png" alt="cat icon" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="images/linkedin.png" alt="linkldn icon" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="images/vector.png" alt="two icon" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="images/twitter-logo.png" alt="twitter icon" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="images/m-logo.png" alt="m icon" /></a>
+          </li>
+        </ul>
+      </div>
+
+      <div class="anchor-arrow">
+        <img src="images/anchor-arrow.png" alt="arrow" />
+      </div>
+    </section>
+    <section class="project-container">
+      <div class="project-title">
+        <h2>My Recent </h2>
+      </div>
+      <div class="project-list">
+        <ul>
+          <li class="list-item">
+            <div class="image-project"></div>
+            <div class="image-style">
+              <h2>Tonic</h2>
+              <div class="buttons">
+                <div>PHP</div>
+                <div>css</div>
+                <div>JavScript</div>
+                <div>html</div>
+              </div>
+              <div class="button-style-link">
+                <button class="button" type="button">Open</button>
+              </div>
+            </div>
+          </li>
+          <li class="list-item">
+            <div class="image-project"></div>
+            <div class="image-style">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+              <div class="buttons">
+                <div>Ruby on rails</div>
+                <div>css</div>
+                <div>JavScript</div>
+                <div>html</div>
+              </div>
+              <div class="button-style-link">
+                <button class="button" type="button">Open</button>
+              </div>
+            </div>
+          </li>
+          <li class="list-item">
+            <div class="image-project"></div>
+            <div class="image-style">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+              <div class="buttons">
+                <div>Ruby on rails</div>
+                <div>css</div>
+                <div>JavScript</div>
+                <div>html</div>
+              </div>
+              <div class="button-style-link">
+                <button class="button" type="button">Open</button>
+              </div>
+            </div>
+          </li>
+        </ul>
+      </div>
+    </section>
+
+    <div class="about-me">
+      <section class="about-container">
+        <header class="about-header">
+          <h2>About me</h2>
+        </header>
```

Comment:
- [ ] This is looking good, but let's leave only one `header` tag at the top of the document, this is usually better for semantics. You can replace it with a `div` tag and it won't affect your design.